Userland/Libraries/LibVideo/VP9/Decoder.cpp

```diff
@@ -560,10 +560,328 @@ DecoderErrorOr<void> Decoder::predict_intra(u8 plane, u32 x, u32 y, bool have_le
     return {};
 }
 
-DecoderErrorOr<void> Decoder::predict_inter(u8, u32, u32, u32, u32, u32)
+MotionVector Decoder::motion_vector_selection(u8 plane, u8 ref_list, u32 block_index)
```

Comment:
`select_motion_vector` ? you converted "the motion vector clamping process" to `clamp_motion_vector` below. either way they should be consistent.

Reply:
Ah, good point, I didn't notice the inconsistency when I was writing this. It helps that the section name is commented so it can still be found in spec.

That also reminds me, clamp_motion_vector is named very similarly to clamp_mv (spec name), I'll add a FIXME to rename later if it makes sense.